tests/serialisation_tests/test_serialisation.py

```diff
@@ -221,6 +222,23 @@ def test_serialise_download_data_no_filters(seeded_test_client, additional_test_
         "Place",
         "OrganisationName",
     ]
+    assert list(test_serialised_data["ProjectFinanceChange"][0].keys()) == [
```

Comment:
out of interest, if we don't add PFC data in the conftest, does this return here as an empty table, or do we just drop empty tables?

Reply:
It returns an empty table.

The current behaviour is that we serialise out empty tabs where there's no information for a given table relative to a set of queries. I presumed this would remain the case for empty tables, or else we'll have to write conditionals for tables that we don't want to serialise out if empty.